### DISCUSSION TITLE
图标显示为问号，需要安装什么字体么？

### DISCUSSION THREAD

**Discussion by aztack** [nshen/learn-neovim-lua#73] [General]
MacOS
![image](https://user-images.githubusercontent.com/782871/183411233-9e4eeef8-9020-4119-8de3-f77a160ea70b.png)

**Comment by nshen** [comment]
nerdfont 安装方法小册里有介绍呀

**Reply by chernolll** [reply]
请问我是在服务器上安装nerdfonts的，安装完后，网上都是在图形界面里改字体；没有图形界面的要如何更改呢

**Reply by nshen** [reply]
我觉得server上不需要安装nerdfonts，应该在有图形界面的本地安装nerdfonts 然后ssh到server。

**Reply by chernolll** [reply]
本地终端ssh连接远程后，还是问号符号（本地有nerdfonts字体了），有办法通过命令行去实现图形化界面更改字体的方法吗（困扰很久了）

**Reply by nshen** [reply]
我试了一下好像不会呀，本地配置好后 `nvim .`如果不显示❓了，那么ssh xx@xx 后 `nvim .` 也不会显示问号呀，服务器端没装字体呀

**Comment by aztack** [comment]
```bash
brew tap homebrew/cask-fonts
brew install --cask font-hack-nerd-font
```

<img width="943" alt="image" src="https://user-images.githubusercontent.com/782871/183800376-5f44bdf6-a5dc-4a67-baea-8796ecdbe483.png">

**Comment by chernolll** [comment]
您好，我是魏健，我已收到您的邮件！
